Client/mods/deathmatch/logic/luadefs/CLuaVehicleDefs.cpp

```
@@ -4340,3 +4342,19 @@ bool CLuaVehicleDefs::SpawnVehicleFlyingComponent(CClientVehicle* const vehicle,
 
     return vehicle->SpawnFlyingComponent(partNodeIndex, collisionType, removalTime.value_or(-1));
 }
+
+bool CLuaVehicleDefs::SetSmokeTrailEnabled(CClientVehicle* vehicle, bool state) noexcept
+{
+    std::uint16_t model = vehicle->GetModel();
+    if (model != 512 && model != 513)
+        return false;
```

It's better to throw an exception here instead of silent false.